fastlane/Fastfile

```diff
@@ -70,11 +70,54 @@ platform :android do
       validate_only: false
     )
 
-    cleanup_fastlane_release_notes()
+    flversion = convert_version(release_number: version)
+    cleanup_fastlane_release_notes(release_number: flversion)
```

Comment:
passing version code in so that `cleanup_fastlane_release_notes` doesn't have to recalculate it based on the value on disk.